soc/riscv/nordic_nrf/common/vpr/Kconfig

```diff
@@ -14,5 +14,6 @@ config RISCV_CORE_NORDIC_VPR
 	select RISCV_ISA_EXT_ZIFENCEI
 	select RISCV_SOC_HAS_ISR_STACKING
 	select RISCV_SOC_CONTEXT_SAVE
+	select HAS_FLASH_LOAD_OFFSET if XIP
```

Comment:
Not sure that it needs to be `if XIP`.

Reply:
I guess it doesn't hurt if always enabled?